### DISCUSSION TITLE
Add support for historical exchange rates?

### DISCUSSION THREAD

**Discussion by malle-pietje** [worksome/exchange#22] [Ideas]
Like the package, thanks for spending time on maintaining it!

Are there any plans to implement support for **historical** rates? I already directly use the ExchangeRate.host API and the Frankfurter.app API in one of my projects where we fetch historical rates when needed. It's not that complex, both APIs even use the same date format.

If your package supports historical rates I can drop my own custom implementation, I expect others might also be interested in such a feature.

**Comment by jadessoriano** [comment]
Up, @malle-pietje  did u extend this package? may I ask on how you did the implementation to support this feature?
uhm, nvm, figured it out. would still be nice for this to be a feature tho
